Soccar [UI]/src/nl/soccar/ui/fx/drawable/MapUiFx.java

```diff
@@ -29,8 +26,6 @@
     private static final float WALL_WIDTH;
     private static final float CORNER_SIZE;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "CORNER_SIZE" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1068)